src/consensus/multiera.md

```diff
@@ -0,0 +1,57 @@
+# Multi-era considerations
```

Comment:
Shouldn't this be part of an ADR? Maybe we could simply explain the approach Cardano takes.